templates/CRM/common/jcalendar.tpl

```diff
@@ -18,9 +18,9 @@
     {$form.$elementName.$elementIndex.html}
 {elseif $blockId and $blockSection}
     {assign var='elementId'   value=$form.$blockSection.$blockId.$elementName.id}
-    {assign var="tElement" value=`$elementName`_time}
+    {assign var="tElement" value="`$elementName`_time"}
```

Comment:
Would be nice if we could just delete this, but still used in a bunch of extensions, nowhere in core.

Reply:
@agileware-justin  - looks like eway recurring is one of those places - looks lke you don't need it if you have

``$this->add('datepicker', ...)` instead of the deprecated `addDateTime()`